<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio_sesion</class>
 <widget class="QMainWindow" name="inicio_sesion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(197, 220, 247, 1), stop:1 rgba(255, 255, 255, 1));</string>
  </property>
  <widget class="QWidget" name="widget_fondo">
   <widget class="QWidget" name="widget_menu" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>430</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDE2EA;
border: 2px solid #243769;
border-radius: 10px;</string>
    </property>
    <widget class="QLabel" name="lab_iniciar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial Black&quot;;
background-color: #9CD0DC;
border: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#243769;&quot;&gt;INICIAR SESIÓN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="linea">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>341</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
border: 2 px solid #FFE384;
border-radius: 10px;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lab_registro">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial Black&quot;;
background-color: #BDE2EA;
border: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#243769;&quot;&gt;REGISTRARSE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_paisaje">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>182</y>
       <width>428</width>
       <height>340</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(&quot;C:/Program Files (x86)/Qt Designer/projects/Red de Servicios/iconos/paisaje.jpg&quot;);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lab_salir">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FD9390;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;❌&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>381</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout">
     <item>
      <widget class="QStackedWidget" name="stacked">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(197, 220, 247, 1), stop:1 rgba(204, 228, 255, 1));</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_iniciar">
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF;
border: 2px solid #9CC0F6;
border-radius: 10px;</string>
        </property>
        <widget class="QLabel" name="lab_nombre">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>200</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #FFFFFF;
border: 0px solid #DADADA;
border-radius: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#2b9699;&quot;&gt;Nombre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_nombre">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>240</y>
           <width>301</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #E8EEF2;
font: 9pt &quot;Verdana&quot;;
border: 1px solid #DADADA;
border-radius: 1px;
color: #3A3A3A;</string>
         </property>
        </widget>
        <widget class="QLabel" name="lab_contra">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>300</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #FFFFFF;
border: 0px solid #DADADA;
border-radius: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#2b9699;&quot;&gt;Contraseña&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_contra">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>340</y>
           <width>301</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #E8EEF2;
font: 9pt &quot;Verdana&quot;;
border: 1px solid #DADADA;
border-radius: 1px;
color: #3A3A3A;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_acceder">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>430</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #A3F4ED;
border: 2px solid #2B9699;
border-radius: 5px;
color: #2B9699;</string>
         </property>
         <property name="text">
          <string>ACCEDER</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_usuario">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>140</width>
           <height>140</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(&quot;C:/Program Files (x86)/Qt Designer/projects/Red de Servicios/iconos/usuario.png&quot;);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_registro">
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF;
border: 2px solid #9CC0F6;
border-radius: 10px;</string>
        </property>
        <widget class="QLabel" name="lab_nueva_contra">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>300</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #FFFFFF;
border: 0px solid #DADADA;
border-radius: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#2b9699;&quot;&gt;Crear nueva contraseña&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_crear">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>430</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #2B9699;
border: 2px solid #A3F4ED;
border-radius: 5px;
color: #A3F4ED;</string>
         </property>
         <property name="text">
          <string>CREAR</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_nuevo_nombre">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>240</y>
           <width>301</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #E8EEF2;
font: 9pt &quot;Verdana&quot;;
border: 1px solid #DADADA;
border-radius: 1px;
color: #3A3A3A;</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_nueva_contra">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>340</y>
           <width>301</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #E8EEF2;
font: 9pt &quot;Verdana&quot;;
border: 1px solid #DADADA;
border-radius: 1px;
color: #3A3A3A;</string>
         </property>
        </widget>
        <widget class="QLabel" name="lab_nuevo_nombre">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>200</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial Black&quot;;
background-color: #FFFFFF;
border: 0px solid #DADADA;
border-radius: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#2b9699;&quot;&gt;Crear nuevo nombre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_usuario_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>140</width>
           <height>140</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(&quot;C:/Program Files (x86)/Qt Designer/projects/Red de Servicios/iconos/agregar-usuario.png&quot;);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
